.circleci/config.yml

```diff
@@ -146,7 +146,7 @@ jobs:
                     -secondary-kubecontext="kind-dc2" \
                     -debug-directory="$TEST_RESULTS/debug" \
                     -consul-image="hashicorp/consul-enterprise:1.10.0-ent-beta2" \
-                    -consul-k8s-image="docker.mirror.hashicorp.services/hashicorpdev/consul-k8s:latest"
+                    -consul-k8s-image="ishustava/consul-k8s-dev:05-14-2021-d7662f1"
```

Comment:
I think you can change the image back right before merging since the k8s changes are merged

Reply:
yep!